views/search.handlebars

```diff
@@ -41,7 +71,7 @@
                 </button>
             </div>
             <div class="modal-body" id="modalBody">
-
+                Are you sure you want this book?: <br><b><p id="requestTitle"></p></b>
```

Comment:
Bold tags are not bootstrap. The p tag should have like class="font-weight-bold" on it